src/usr/local/www/services_captiveportal.php

```diff
@@ -633,13 +651,19 @@ function build_authserver_list() {
 	))->setHelp("If enabled, connected users won't be disconnected during a pfSense reboot.");
 }
 
-$section->addInput(new Form_Checkbox(
+$concurrentlogins_list = array("disabled" => "Disabled", "multiple" => "Multiple", "last" => "Last login", "first" => "First login");
```

Comment:
Define this up higher, then you can also use the keys when validating rather than having to duplicate that content with another array. Then if it changes in the future, only one place needs updating.

Also, the user-visible parts should be run through `gettext()`.